tests/run-tests-on-external-cluster.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash -e
+
+: {$KUBEVIRTNAMESPACE:="openshift-cnv"}
+_kubectl=$(which kubectl)
+_oc=$(which oc)
+
+[[ -z ${_kubectl} && -z ${_oc} ]] && echo "OC or KUBECTL is not installed" && exit 1
+
+[ -z "${KUBECONFIG}" ] && echo "KUBECONFIG is not set" && exit 1
+
+cat << EOF > test_config.json
+{
+ "storageClassLocal": "hostpath-provisioner",
+ "storageClassHostPath": "hostpath-provisioner",
+ "storageClassRhel": "hostpath-provisioner",
+ "storageClassWindows": "hostpath-provisioner",
+ "manageStorageClasses": false
+}
+EOF
+
+oc create namespace kubevirt
+rm -rf _out
+hack/dockerized hack/build-func-tests.sh
+make manifests
+rm -f _out/manifests/testing/cdi-*
+rm -f _out/manifests/testing/kubevirt-config.yaml
+sed -i "s/namespace: kubevirt/namespace: ${KUBEVIRTNAMESPACE}/g" _out/manifests/testing/*.yaml
+_out/tests/tests.test --cdi-namespace=${KUBEVIRTNAMESPACE} \
+	--config=${PWD}/test_config.json \
+	--container-tag=v0.26.1 --container-tag-alt=v0.26.1_alt \
```

Comment:
@phoracek @EdDev We can remove container-tag and container-tag-alt since these flags aren't mandatory. In general, they are used in virt-operator custom installation tests. If we want to give the user the ability to pass these flags we need to update the script to parse these parameters when the script is called. If these flags aren't defined the tests will be skipped. [link](https://github.com/kubevirt/kubevirt/blob/d38e5acc3c7773bdd19fe4b73ea07477af06860e/tests/operator_test.go#L938)

Reply:
Thanks, removing these.